tests/unit/components/ContentNode/TabNavigator.spec.js

```diff
@@ -69,4 +69,49 @@ describe('TabNavigator.spec', () => {
     expect(wrapper.find('.tabs-content').text()).toBe('Second');
     expect(tabnav.props('value')).toEqual(titles[1]);
   });
+
+  it('selects correct tab when adding a tab', async () => {
+    const longerTitles = ['Long tab title',
```

Comment:
Something like this might help save some typing if you need to do similar things in the future:

`const longerTitles = titles.concat('added title');`

So you don't have to copy the full array in each test.